src/components/Calendar.jsx

```diff
@@ -165,45 +158,69 @@ const Calendar = ({ history }) => {
           <div
             className={`col cell ${
               !dateFns.isSameMonth(day, monthStart)
-                ? "disabled"
-                : dateFns.isSameDay(day, selectedDate) ? "selected" : ""
-              }`}
+                ? 'disabled'
+                : dateFns.isSameDay(day, selectedDate)
+                ? 'selected'
+                : ''
+            }`}
             key={day}
             onClick={() => onDateClick(dateFns.format(cloneDay, 'yyyy-MM-dd'))}
           >
-            <span className="number">{formattedDate}</span>
+            <span className='number'>{formattedDate}</span>
             {/* <span className="bg">{formattedDate}</span> */}
```

Comment:
This commented line can come out, right?

Reply:
I removed this line from from inside the branch i'm working on currently (feature/screenshot-button) so it will be gone when that merges eventually.